core/workspace.js

```diff
@@ -415,6 +415,7 @@ Blockly.Workspace.prototype.deleteVariableById = function(id) {
  * @param {!Blockly.VariableModel} variable Variable to delete.
  * @param {!Array.<!Blockly.Block>} uses An array of uses of the variable.
  * @private
+ * @suppress {unusedPrivateMembers}
  */
 Blockly.Workspace.prototype.deleteVariableInternal_ = function(variable, uses) {
```

Comment:
Why did you keep this function around?

Reply:
It wasn't clear to me whether it was one of those APIs that is set as private but is actually used, and if we wanted to deprecate before removing.
Based on your comment, it sounds like you think this can be removed.

Reply:
I just took a look at the history and we're good to remove it :+1:

Reply:
Done.